app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="col-lg-3 mx-lg-auto m-2">
+<div class="card shadow">
+<h2 class="card-header text-center">Log in</h2>
+<div class="card-body">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="form-group">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class:"form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", class:"form-control" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="form-check">
+      <%= f.check_box :remember_me , class:"form-check-input"%>
+      <%= f.label :remember_me, class:"form-check-label" %>
```

Comment:
- [ ] Great job working on the login with devise but it would be great if you made the styles the same given design since in the requirements you should stick with the design layout as much as possible so I could see in your design is different than the given design so please change it to be the same with the nav as shown ⬇🔥💯🤩
![image](https://user-images.githubusercontent.com/58553711/235317919-3203e23b-2b32-4bd5-a0e2-1e077d26bdd2.png)



- Your
![image](https://user-images.githubusercontent.com/58553711/235317872-cb122a0c-8903-4f2b-9f3e-a26d864402b8.png)


- Given
![image](https://user-images.githubusercontent.com/58553711/235317899-aa55908f-87a9-4ce2-92c6-2c8c35640317.png)